modin/pandas/dataframe.py

```diff
@@ -177,12 +189,6 @@ def __repr__(self):
             return result
 
     def _repr_html_(self):  # pragma: no cover
```

Comment:
I think a docstrings should be kept for private methods.

Reply:
In general, I agree, but specifically for this method, the documentation is copied from the dataframe pandas.

Reply:
I think something goes wrong because docstring for the method in pandas is the following:
```python
"""
Return a html representation for a particular DataFrame.

Mainly for IPython notebook.
"""
```

Reply:
The same problem relates to other methods.

Reply:
I don't see what the problem is.
Can you clarify this?

Reply:
I don't understand a little bit. Why are some methods' docstrings deleted, some methods' docstrings changed, some methods' docstrings added? I supposed we are keeping our docstrings and just change its style.

Reply:
The reason I removed the dockstrings is because manually adding dockstrings for public methods that copy the behavior of pandas makes it difficult to transition to support for the new Pandas release. Because during the transition, we will need to look not only at the equivalence of behavior in the code, but also at these very docking strings that the pandas community could have changed, otherwise there may be a discrepancy between what is written in the documentation and what will actually happen by calling that or some other function.

Reply:
Almost everything else relates to `numpydoc` requirements.

Reply:
I see, it makes sense.